sqlglot/parser.py

```diff
@@ -1147,6 +1147,13 @@ def _parse_table(self, schema=False):
         if schema:
             return self._parse_schema(this=this)
 
+        while True:
```

Comment:
this should go in query modifiers

Reply:
line 1022

Reply:
I originally added it there, but PIVOT goes before the table alias. Query modifiers are parsed after the alias.

Reply:
before table alia’s!? in that case, you don’t need to add it to query modifiers at all like you did in expressions.py

Reply:
it may also be more similar to a table sample as well which is a nested object

Reply:
i don’t think it goes before alia’s, i think it has its own alias

Reply:
can you do select * from a pivot() b